plugins/fs/fs.c

```diff
@@ -106,13 +122,26 @@ run(LuastatusPluginData *pd, LuastatusPluginRunFuncs funcs)
     while (1) {
         // make a call
         lua_State *L = funcs.call_begin(pd->userdata);
-        lua_createtable(L, 0, p->paths.size);
+        lua_newtable(L);
```

Comment:
Why this change? There will be at least `p->paths.size` hash entries in the table.

Reply:
Well, I just thought that since the number of glob matches may be anything, it is not that useful to preallocate a fixed number of entries. But, I think you are right -- it won't hurt if we do.